kaguya/scripts/script.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+$('a:not([href^="#"]):not([target])').on('click', (e) => {
+  e.preventDefault();
+  $('body')
+    .on('transitionend', () => {
+      window.location = e.currentTarget.href;
+    })
+    .css('opacity', 0.1);
+});
+
+$('#header-nav-button').on('click', () => {
+  $('.header-nav').slideToggle();
+  $('.nav-button-line').toggleClass('active');
+});
+
+$('.main-news-list li').hover(function () {
+  if ($(window).width() > 375) {
+    $(this).find('.main-news-background').slideToggle();
+    $(this).find('figcaption').toggleClass('active');
+  }
+});
```

Comment:
(修正不要)
参考までに、ここもfunctionキーワードをアロー関数に置き換えた方がよいかなってとこです。

[アロー関数のみにしていく](https://qiita.com/shibukawa/items/19ab5c381bbb2e09d0d9)

```suggestion
$('.main-news-list li').hover((e) => {
  if ($(window).width() > 375) {
    $(e.currentTarget).find('.main-news-background').slideToggle();
    $(e.currentTarget).find('figcaption').toggleClass('active');
  }
});
```